### PR TITLE
autoid_service: setup OWNERS file for reviewing assigning

### PR DESCRIPTION
**Pull request by wuhuizuo**
Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### What problem does this PR solve?

Issue Number: ref #46911

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code
  > - [x] I checked and no code files have been changed.

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by wuhuizuo** [comment]
/cc @bb7133 
PTAL

**Comment by wuhuizuo** [comment]
/hold

It depends on bot's new deployments, when it's ready @wuhuizuo will unhold it.

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -8,3 +8,11 @@ aliases:
   sig-critical-approvers-parser:
     - yudongusa
     - easonn7
+  sig-approvers-autoid-service: # approvers for auto-id service
```
Comment (bb7133) [inline OWNERS_ALIASES:11]: Maybe we can just need 1 owner only: @tiancaiamao

Comment (wuhuizuo) [reply OWNERS_ALIASES:11]: you can update it by suggestion codes.

> Maybe we can just need 1 owner only: @tiancaiamao

Comment (wuhuizuo) [reply OWNERS_ALIASES:11]: It is best to set up 2 or more members.

**Comment by wuhuizuo** [comment]
/lgtm 

for test

**Comment by wuhuizuo** [comment]
/unhold

Bot deployment is refreshed.

**Review by tiancaiamao — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**